deploy/cephcsi/image/Dockerfile

```diff
@@ -8,14 +8,14 @@ LABEL stage="build"
 
 ARG CSI_IMAGE_NAME=quay.io/cephcsi/cephcsi
 ARG CSI_IMAGE_VERSION=canary
-ARG GO_ARCH
```

Comment:
This should not be removed as its a docker build argument

Reply:
It is there 2x, this is a duplicate from the above